view/preview/index.js

```diff
@@ -123,7 +125,10 @@ circles.map((circle) => svg.prepend(circle));
 // add onScroll logic
 useScroll({
   element: document.querySelector(SELECTORS.WRAPPER),
-  onStartScroll: timeline.pause,
+  onStartScroll: () => {
+    timeline.pause();
+    timeline.seek(timeline.duration * (timeline.progress * 0.01) + 10);
```

Comment:
Hack)

Our flow:

1. We autostart animation.
2. If we start to scroll:
2.1 Pause animation
2.2 Mix animation progress + scroll position on scroll
2.3 When scroll is finished, play the animation

The problem in the step 2.2. We have a bit incorrect mix of animation progress and scroll position because library don't have an opportunity to work with it. 

I tried solution with scroll diff and several other ideas but this huck with custom inertion value works more smooth.